src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -75,16 +75,15 @@
             List<InstructorAttributes> instructorsToRemind, List<InstructorAttributes> instructorsToNotify) {
 
         CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
-        String template = EmailTemplates.USER_FEEDBACK_SESSION;
+        String template = EmailTemplates.USER_FEEDBACK_SESSION
+                .replace("${status}", "is still open for submissions");
```

Comment:
Since you are changing this, put `"is still open for submissions"` in EmailTemplateMessages too. For future PRs, do try to avoid making unrelated changes.